sdk/test-utils/recorder/src/utils.ts

```diff
@@ -12,7 +12,15 @@ export function isRecordMode() {
   return env.TEST_MODE === "record";
 }
 
+export function isLiveMode() {
+  return env.TEST_MODE === "live";
+}
+
 export function isPlaybackMode() {
+  if (!(isRecordMode() || isLiveMode())) {
```

Comment:
Can this not be `return (!isRecordMode() && !isLiveMode())` instead?

Reply:
I'm just being extra cautious and setting the env.TEST_MODE="playback".
I remember Daniel checking `env.TEST_MODE==="playback"` in keyvault tests and me doing the same for storage in some instances. 
I had updated the storage with `isPlaybackMode()` instead.
.
.
I can do your suggestion too, but just we just need to make sure that we update everywhere.

Reply:
None of the packages should be checking the TEST_MODE directly anymore. They should use the helper methods defined here.

Reply:
None of the packages should be checking the TEST_MODE directly anymore. They should use the helper methods defined here.

Reply:
Yes, omw, updating everywhere.